.github/workflows/regression-tests.yml

```diff
@@ -325,3 +324,42 @@ jobs:
           -n "${{matrix.extra_name}}${{matrix.sim}} (${{matrix.sim-version}}) | ${{matrix.os}} | Python ${{matrix.python-version}}" \
           -e SIM,TOPLEVEL_LANG,CXX,OS,PYTHON_VERSION \
           -a "-rl"
+
+  mingw-env:
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - sim: verilator
+            lang: verilog
+            pkg: verilator
+            may_fail: true
+          - sim: icarus
+            lang: verilog
+            pkg: iverilog
+          - sim: ghdl
+            lang: vhdl
+            pkg: ghdl-llvm
```

Comment:
You might use `ghdl`, instead of `ghdl-llvm`. That should work on either MINGW32 or MINGW64.

Are you interested in testing on MINGW32 or cocotb is only expected to be used on 64 bits?

Reply:
> Are you interested in testing on MINGW32

Not unless I have to. Our regressions are long enough already.

> is only expected to be used on 64 bits?

We still have to deal with the 32 bit free version of Modelsim. Until that goes away for good we need to care about 32 bit support. That being said, we have 0 users using MINGW right now. The reason for adding MINGW at all is because their Python setup is weird, I put in effort to get it to work, and I'd like to track our support of that environment.

Reply:
Are you still interested in packaging cocotb for MINGW?

Reply:
Fair enough.

For the record, in https://github.com/hdl/MINGW-packages/actions/runs/1080755117 I'm running cocotb with GHDL and iverilog on MINGW64 and MING32. That is a canary workflow though, not something I check every day.

Reply:
> Are you still interested in packaging cocotb for MINGW?

Yes. However, I did not package any Python project yet. I need to check/learn what is specific compared to packagin Ada/C/C++.